packages/clerk-js/src/utils/authPropHelpers.ts

```diff
@@ -112,3 +112,13 @@ export const buildAuthQueryString = (data: BuildAuthQueryStringArgs): string | n
   }
   return Object.keys(query).length === 0 ? null : qs.stringify(query);
 };
+
+export const normalizeRoutingOptions = ({
+  routing,
+  path,
+}: {
+  routing?: RoutingStrategy;
+  path?: string;
+}): RoutingOptions => {
+  return { routing: !!path && !routing ? 'path' : routing, path } as RoutingOptions;
```

Comment:
Let's make this more verbose. 
In general, one-liners are fine, but in this case, expanding the ternary makes sense so we can have one block for each possible scenario, even if it's not completely necessary.

Reply:
Done!